dev/com.ibm.websphere.appserver.features/visibility/public/mpFaultTolerance-4.1/io.openliberty.mpFaultTolerance-4.1.feature

```diff
@@ -0,0 +1,26 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpFaultTolerance-4.1
+visibility=public
+singleton=true
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-API-Package: org.eclipse.microprofile.faulttolerance.exceptions;  type="stable", \
+                 org.eclipse.microprofile.faulttolerance;  type="stable"
+IBM-ShortName: mpFaultTolerance-4.1
+Subsystem-Name: MicroProfile Fault Tolerance 4.1
+# io.openliberty.mpCompatible-5.0; ibm.tolerates:="6.0,6.1" comes from io.openliberty.mpConfig features
+-features=io.openliberty.mpConfig-3.0; ibm.tolerates:="3.1", \
+  io.openliberty.org.eclipse.microprofile.faulttolerance-4.0, \
+  io.openliberty.cdi-3.0; ibm.tolerates:="4.0"
```

Comment:
This feature is new in MP 7 and should only be compatible with other features in MP 7.

That probably means getting the right version of all the dependencies, removing the tolerates and having your own dependency on mpCompatible-7.0.

Reply:
#28463 Open for this